packages/components/src/form-file-upload/types.ts

```diff
@@ -10,6 +10,12 @@ import type Icon from '../icon';
 
 // TODO: Replace `children` and `icon` types with props from Button once Button is typed.
 export type FormFileUploadProps = {
+	/**
+	 * Start opting into the larger default height that will become the default size in a future version.
+	 *
+	 * @default false
+	 */
+	__next40pxDefaultSize?: boolean;
```

Comment:
This was passed through as rest props to the underlying Button, but adding it here so it shows up explicitly in Storybook.